eslint-plugin-expensify/CONST.js

```diff
@@ -13,8 +13,8 @@ module.exports = {
         PREFER_STR_METHOD: 'Prefer \'Str.{{method}}\' over the native function.',
 
         // eslint-disable-next-line max-len
-        NO_MULTIPLE_API_CALLS: 'Do not call API or DeprecatedAPI multiple times in the same method. The API response should return all the necessary data in a single request, and API calls should not be chained together.',
-
+        NO_MULTIPLE_API_CALLS: 'Do not call API multiple times in the same method. The API response should return all the necessary data in a single request, and API calls should not be chained together.',
+        NO_CALL_ACTIONS_FROM_ACTIONS: 'Calling actions from inside other actions is forbidden. If an action needs to call another action combine the two actions into a singular API call instead.',
```

Comment:
Still need to test this against `Expensify/App` - but is this going to be a pain? We have a lot of "actions" called from "other actions" because the actions are not well defined.